dev/conversation/styles.scss

```diff
@@ -267,7 +266,11 @@
     .vjs-video-wrap,
     .video-js {
       max-width: 100%;
-      height: 400px;
+      height: auto;
+    }
+
+    .video-js .vjs-tech {
+      position: static;
```

Comment:
Remove it if it is default

Reply:
I can't it doesn't work without it.

Reply:
Looks like this setting comes from outside.

Reply:
It sets it to absolute by default.